mlflow/metrics/genai/metric_definitions.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Optional
+
+from mlflow.exceptions import MlflowException
+from mlflow.metrics.base import EvaluationExample
+from mlflow.metrics.genai.make_genai_metric import make_genai_metric
+from mlflow.metrics.genai.utils import _get_latest_metric_version
+from mlflow.models import EvaluationMetric
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, INVALID_PARAMETER_VALUE
+from mlflow.utils.class_utils import _get_class_from_string
+
+DEFAULT_MODEL = "openai:/gpt4"
```

Comment:
Can we get the default model from here as well?
https://github.com/mlflow/mlflow/pull/9673/files#diff-6f989efdf75d569970663449be1700307f6889952028f66ad632cfd7e47bc241R12